tdrs-backend/tdpservice/settings/common.py

```diff
@@ -335,7 +335,7 @@ class Common(Configuration):
     CSP_SCRIPT_SRC = ("'self'", s3_src)
     CSP_IMG_SRC = ("'self'", "data:", s3_src)
     CSP_FONT_SRC = ("'self'", s3_src)
-    CSP_CONNECT_SRC = ("'self'", "*.cloud.gov")
+    CSP_CONNECT_SRC = ("'self'", "*.cloud.gov", "localhost:*")
```

Comment:
can this addition be moved to `settings/local.py`? i don't see it overridden in `settings/cloudgov.py` which seems to imply the CSP would allow connections from `localhost` in deployed/prod environments as well